### PR TITLE
Beneficiary: added DOB, region, address, connection with Beneficiary

### PR DESCRIPTION
**Pull request by mayapeneva**

### CODE REVIEW

File: protos/campaign.proto [hunk 1]
```diff
@@ -78,13 +78,17 @@ message CampaignTypesResponse {
...
-  common.nomenclatures.BeneficiaryType type = 3;
-  string organizer_id = 4;
-  common.nomenclatures.CountryCode country_iso_code = 5;
-  common.nomenclatures.City city = 6;
-  string email = 7;
-  string phone = 8;
-  string website = 9;
+  google.protobuf.Timestamp date_of_birth = 3;
+  common.nomenclatures.BeneficiaryType type = 4;
+  string organizer_id = 5;
```
Comment (imilchev) [inline protos/campaign.proto:83]: I see the ordering of the properties has changed. This is not an issue as long as nothing is using that message apart from the campaign modules. Is that the case?

Comment (mayapeneva) [reply protos/campaign.proto:83]: Campaign.proto is used by campaigns module only I think. 
Should I add the new fields at the end, not among the other fields?

Comment (preslavgerchev) [reply protos/campaign.proto:83]: Protobuf makes use of the number to indicate which field represents what when in a binary format. If someone is using an older definition of the `campaign.proto` they'd expect that `5` is the country code whereas on our side we will parse this as an `organizer_id` - generally new fields indeed go at the end

File: protos/common/nomenclatures.proto [hunk 2]
```diff
@@ -286,187 +286,23 @@ enum City {
...
-  CURRENCY_CODE_XPT = 171;
-  CURRENCY_CODE_XSU = 172;
-  CURRENCY_CODE_XTS = 173;
-  CURRENCY_CODE_XUA = 174;
-  CURRENCY_CODE_XXX = 175;
-  CURRENCY_CODE_YER = 176;
-  CURRENCY_CODE_ZAR = 177;
-  CURRENCY_CODE_ZMW = 178;
-  CURRENCY_CODE_ZWL = 179;
+enum Connection {
```
Comment (preslavgerchev) [inline protos/common/nomenclatures.proto:289]: I think this should be in `campaign.proto` - it's pretty unique to a beneficiary and will likely not be used by the other proto files

Comment (preslavgerchev) [reply protos/common/nomenclatures.proto:289]: Also, consider renaming to `BeneficiaryConnection` to make it a bit more explicit

Comment (mayapeneva) [reply protos/common/nomenclatures.proto]: Thank you

File: protos/campaign.proto [hunk 3]
```diff
@@ -166,8 +170,31 @@ message Beneficiary {
...
+  BENEFICIARY_CONNECTION_GRANDCHILD = 6;
+  BENEFICIARY_CONNECTION_GRANDPARENT = 7;
+  BENEFICIARY_CONNECTION_WIFE = 8;
+  BENEFICIARY_CONNECTION_HUSBAND = 9;
+  BENEFICIARY_CONNECTION_COUSIN = 10;
+  BENEFICIARY_CONNECTION_AUNT = 11;
+  BENEFICIARY_CONNECTION_UNCLE = 12;
+  BENEFICIARY_CONNECTION_OTHER_RELATIVE = 13;
+  BENEFICIARY_CONNECTION_FRIEND = 14;
+  BENEFICIARY_CONNECTION_AUNTCONNECTION_OTHER = 15;
```
Comment (preslavgerchev) [inline protos/campaign.proto:199]: Think these are supposed to be 2 values

**Review by preslavgerchev — CHANGES_REQUESTED** [review]

File: protos/campaign.proto (hunk 3 above)
Comment (mayapeneva) [reply protos/campaign.proto:199]: Sure. Thank you.

**Review by preslavgerchev — APPROVED** [review]

**Pull request merged**